static/src/javascripts/projects/common/modules/preferences/main.js

```diff
@@ -0,0 +1,69 @@
+define([
+    'react',
```

Comment:
Yep that makes sense. I'm not against micro-libs or anything, just think dead code elimination would probably give us more benefit in the end, as there'll be a whole bunch of code we never really use that gets included in each build.